packages/components/src/components/Calculator/calculatorReducer.ts

```diff
@@ -27,14 +27,9 @@ export type CalculatorState = {
 
 // This function is used to determine if a calculator has changed.
 // It's obviously not perfect - it doesn't capture changes within the calculator function, but this would be much more complicated.
```

Comment:
Remove this comment too? SqCalculator is responsible for that now and you copied the comment there.